src/main/java/id/co/veritrans/mdk/v1/VtGatewayConfigBuilder.java

```diff
@@ -12,6 +12,8 @@
     private String clientKey;
     private int maxConnectionPoolSize = 16;
     private ProxyConfig proxyConfig;
+    private int connectTimeout = VtGatewayConfig.DEFAULT_CONNECT_TIMEOUT;
```

Comment:
on this part, we can just put directly the default value like `maxConnectionPoolSize`

`private int connectTimeout = 5000;`
`private int socketTimeout = 30000;`